src/libAtomVM/externalterm.c

```diff
@@ -69,9 +71,9 @@
 
 static term parse_external_terms(const uint8_t *external_term_buf, size_t *eterm_size, bool copy, Heap *heap, GlobalContext *glb);
 static int calculate_heap_usage(const uint8_t *external_term_buf, size_t remaining, size_t *eterm_size, bool copy);
-static size_t compute_external_size(term t, GlobalContext *glb);
-static int externalterm_from_term(uint8_t **buf, size_t *len, term t, GlobalContext *glb);
-static int serialize_term(uint8_t *buf, term t, GlobalContext *glb);
+static size_t compute_external_size(term t, bool allow_latin1_encoding, GlobalContext *glb);
```

Comment:
Let's avoid using booleans for flags, let's rather use an enum defining external term opts (e.g. `ExternalTermOpts opts`, so we can pass a `ExternalTermAllowLatin1` or something like that).

booleans are still the choice for predicates.

Reply:
Sounds good.  Added a new field for latin1 encoding.